assignment.rb

```diff
@@ -0,0 +1,103 @@
+# Accesses the csv file with user info
+require "csv"
+
+#Creates a User class with appropriate methods
+class User
+  attr_accessor :name, :pin
+  
+  def initialize(name, pin, balance)
+    @name = name
+    @pin = pin
+    @balance = balance
+  end
+
+  def name
+    @name
+  end
+
+  def pin
+    @pin
+  end
+
+  def balance
+    @balance.to_i
+  end
+
+  def check_balance
+    puts "Your current balance is #{balance}"
+  end
+
+  def withdraw
+    @balance.withdraw_amount
+  end
+end
+
+# Creates a new hash from the csv information
+accounts = []
+csv = CSV.foreach('users.csv', :headers => true, :header_converters => :symbol) { |row|
```

Comment:
also, when you have something on more than 1 line, try to not use `{` and `}` and instead use `do` and `end`